src/java/org/apache/cassandra/tools/nodetool/RebuildIndex.java

```diff
@@ -17,26 +17,38 @@
  */
 package org.apache.cassandra.tools.nodetool;
 
-import static com.google.common.base.Preconditions.checkArgument;
-import static com.google.common.collect.Iterables.toArray;
-import io.airlift.airline.Arguments;
-import io.airlift.airline.Command;
-
 import java.util.ArrayList;
 import java.util.List;
 
 import org.apache.cassandra.tools.NodeProbe;
-import org.apache.cassandra.tools.NodeTool.NodeToolCmd;
+import org.apache.cassandra.tools.nodetool.layout.CassandraUsage;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Parameters;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.collect.Iterables.toArray;
+import static org.apache.cassandra.tools.nodetool.CommandUtils.concatArgs;
 
 @Command(name = "rebuild_index", description = "A full rebuild of native secondary indexes for a given table")
-public class RebuildIndex extends NodeToolCmd
+public class RebuildIndex extends AbstractCommand
 {
-    @Arguments(usage = "<keyspace> <table> <indexName...>", description = "The keyspace and table name followed by a list of index names")
-    List<String> args = new ArrayList<>();
+    @CassandraUsage(usage = "<keyspace> <table> <indexName...>", description = "The keyspace and table name followed by a list of index names")
+    private List<String> args = new ArrayList<>();
+
+    @Parameters(index = "0", description = "The keyspace name", arity = "0..1")
```

Comment:
should keyspace and table be mandatory?

Reply:
https://github.com/apache/cassandra/pull/2497#discussion_r2204598272